.travis.yml

```diff
@@ -1,6 +1,10 @@
 language: node_js
+sudo: false
 node_js:
-  - '0.10'
+  - '6'
+  - '5'
```

Comment:
My understanding of Node's LTS support is that v5 isn't being maintained – is it worth testing against, you think?